### ISSUE TITLE
Announcing: WinUI 3 in Windows App SDK 1.1 Preview 3 ✨

### ISSUE THREAD

**Issue by gabbybilka** [microsoft/microsoft-ui-xaml#7061]
### 1.1 Preview 3 Release of Windows App SDK 

Earlier this week we released our third preview of the 1.1 release of the Windows App SDK! This preview includes new features and bug fixes for WinUI 3 apps, including:
- Mica and Background Acrylic ✨
- New extension categories in MSIX
- Several bug fixes in the Notifications area

To see everything that's new and changed, see the full [release notes here](https://aka.ms/windowsappsdk/1.1/1.1.0-preview3/release-notes). 

### What's next 
To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://aka.ms/winappsdkroadmap), which gets updated regularly.  

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue here, on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

**Comment by michael-hawker** [comment]
Just hit an Access Violation with `ProtectedCursor`, will update to this version and try again to see, currently on 1.0.1

**Comment by michael-hawker** [comment]
Yup still hitting issue with the latest, details here: https://github.com/microsoft/microsoft-ui-xaml/issues/7062

FYI @gabbybilka @ryandemopoulos.

**Comment by JamasChuang94** [comment]
Why I use Nuget package to update WindowsAppSDK to 1.1.0 preview version 3 still can't use Background Acrylic in C++ desktop development

**Comment by michael-hawker** [comment]
Seeing an issue in our CI with the 1.1.0-preview3 package (that I didn't see with 1.0.0, testing 1.0.3 now):

```
Copying file "D:\a\Labs-Windows\Labs-Windows\labs\CiTestExp\tests\CiTestExp.UnitTests.WinAppSdk\bin\x64\Release\net6.0-windows10.0.19041.0\AppxManifest.xml" to layout "AppxManifest.xml"...
Checking whether required frameworks are installed...
Framework: Microsoft.WindowsAppRuntime.1.1-preview3/X86. Skipping installation because framework package architecture does not match target architecture X64 and app package architecture X64.
Framework: Microsoft.WindowsAppRuntime.1.1-preview3/X86. Skipping installation because framework package architecture does not match target architecture X64 and app package architecture X64.
Framework: Microsoft.WindowsAppRuntime.1.1-preview3/X64, app package version 1000.485.2229.0 is not currently installed.
Framework: Microsoft.WindowsAppRuntime.1.1-preview3/ARM64. Skipping installation because framework package architecture does not match target architecture X64 and app package architecture X64.
Installing missing frameworks...
DEP0800: The required framework "C:\Users\runneradmin\.nuget\packages\microsoft.windowsappsdk\1.1.0-preview3\buildTransitive\..\tools\MSIX\win10-x64\Microsoft.WindowsAppRuntime.1.1-preview3.msix" failed to install. [0x80073CF9] Deployment Add operation with target volume C: on Package Microsoft.WindowsAppRuntime.1.1-preview3_1000.485.2229.0_x64__8wekyb3d8bbwe from:  (Microsoft.WindowsAppRuntime.1.1-preview3.msix)  failed with error 0x8000000B. See http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app deployment issues.
	DeploymentSucceeded = False
	DeploymentError = Err_Deploy_SDKInstall
	DeploymentUserError = False
	DeploymentHRESULT = 2147958009
	HasSharedCode = False
	SDKFileName = 
Deleting file "vs.appxrecipe" from layout...
DeployAsync: END (Failure, 0:00:00.402)
```

Is this because it's a preview package or an issue that's regressed?

**Comment by WelterDevelopment** [comment]
How do I actually use Mica? The property "BackdropMaterial.ApplyToRootOrPageBackground" does still not exist in WinAppSDK 1.1 prev3

**Comment by JamasChuang94** [comment]
> 我如何实际使用云母？WinAppSDK 1.1 prev3 中仍然不存在“BackdropMaterial.ApplyToRootOrPageBackground”属性 编辑：好的，我找到了！没有文档，但对 WinUI-Gallery 的拉取请求：[microsoft/WinUI-Gallery@ d9bfb9f](https://github.com/microsoft/WinUI-Gallery/commit/d9bfb9fafd49a6c9ad9201bf3424b9aa02f8d364)
> 
> 我想在运行时更改主题，所以我想出了以下调整：
> 
> SystemBackdropWindow.cs
> MainPage.xaml.cs
> 但是这一切都是那么的复杂！WinAppSDK for Desktop 中是否会有“BackdropMaterial.ApplyToRootOrPageBackground”实现？

It seems that their implementation at this stage will be more complicated than the original version. The official roadmap says that version 1.2 will try to simplify it.

**Comment by knightmeister** [comment]
Can this issue please get addressed - #4804 - surely it's a simple fix, crashing the app when the user right clicks a textbox is absolutely terrible
